File: 242-valid-anagram/242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size()!=t.size())return false;
         sort(t.begin(), t.end()); 
         sort(s.begin(), s.end()); 
        
        
        return s == t;
        
        
    }
};

File: 242-valid-anagram/242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length())
            return false;
        int[] arr = new int[26];
        for(int i=0;i<s.length();i++)
            arr[s.charAt(i) - 'a']++;
        for(int i=0;i<t.length();i++)
            if(arr[t.charAt(i) - 'a']-- == 0)
                return false;
        
        return true;
    }
}